README.md

```diff
@@ -481,6 +482,10 @@ the crystal system.
 | ivQuadField | val | Minimum field for quadratic IV expression | V/m  |
 | ivQuadPower | exp | Exponent: rate = Rate*(E^2-Field^2)^(exp/2) | none |
 
+The keywords l0_e and l0_h are optional. If they are not specified in
+config.txt, they will be computed from other physical constants. If they 
```

Comment:
Can you give the formula here?  I know that plain text, or even Markdown, isn't great for math, but it would good to be specific.

Reply:
I added a formula there. Let me know if the format is okay or if we need to specify what the physical constants are. Or maybe add a reference to Cabrera et al. (2010) paper?